projects/sbb-esta/angular-business/src/lib/header/navbutton/navbutton.component.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostBinding,
+  OnDestroy,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+  selector: 'sbb-navbutton',
```

Comment:
I would like this to be refactored, so the consumer must use this.
Change the selector to 'button[sbbNavbutton]' and refactor the usage.
I realize this is a rather large change, but I believe this reduces complexity.
I apologize for this.

Usage by consumer
```
<sbb-header ...>
  ...
  <button sbbNavbutton type="button" [sbbDropdown]="dropdown">Navigation 4</button>
  ...
</sbb-header>
```

Reply:
No problem, I'll see what I can do to adjust it to that spec.